src/person/simple_serializers.py

```diff
@@ -19,6 +19,16 @@ class SimpleUserSerializer(serializers.ModelSerializer):
     interests = InterestSerializer(source="person.interests", many=True)
     groups = GroupSerializer(source="person.groups", many=True)
     prompts = serializers.SerializerMethodField("get_prompts")
+    pending_feedback = serializers.BooleanField(source="person.pending_feedback")
+    blocked = serializers.SerializerMethodField("get_blocked")
+
+    def get_blocked(self, user):
+        request_user = self.context.get("request_user")
+        if request_user is not None and hasattr(request_user, "person"):
+            return user.id in request_user.person.blocked_users.values_list(
+                "id", flat=True
+            )
```

Comment:
wait so is it multiple user ids?

Reply:
no hehe it's a boolean, this returns true if user.id is in that list and false otherwise

Reply:
LOL okay if it's a boolean you should rename it to `is_blocked_user` or something like that, always have like an `is_` in front